cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityUserOperator.java

```diff
@@ -42,12 +45,16 @@
     public static final String ENV_VAR_ZOOKEEPER_SESSION_TIMEOUT_MS = "STRIMZI_ZOOKEEPER_SESSION_TIMEOUT_MS";
     public static final String ENV_VAR_CLIENTS_CA_CERT_SECRET_NAME = "STRIMZI_CA_CERT_NAME";
     public static final String ENV_VAR_CLIENTS_CA_KEY_SECRET_NAME = "STRIMZI_CA_KEY_NAME";
+    public static final String ENV_VAR_CLIENTS_CA_VALIDITY = "STRIMZI_CA_VALIDITY";
+    public static final String ENV_VAR_CLIENTS_CA_RENEWAL = "STRIMZI_CA_RENEWAL";
 
     private String zookeeperConnect;
     private String watchedNamespace;
     private String resourceLabels;
     private long reconciliationIntervalMs;
     private long zookeeperSessionTimeoutMs;
+    private long clientsCaValidityDays;
+    private long clientsCaRenewalDays;
```

Comment:
These things should not be reconfigured in `Kafka.spec.entityOperator.userOperator`, but rather be taken from `Kafka.spec.clientsCa` implicitly.

Reply:
I am afraid I do not understand :( It takes values from ```result.setClientsCaValidityDays(kafkaAssembly.getSpec().getClientsCa().getValidityDays());
                    result.setClientsCaRenewalDays(kafkaAssembly.getSpec().getClientsCa().getRenewalDays());```
What do you mean by reconfiguring?